app/uk/gov/hmrc/softdrinksindustrylevy/controllers/BalanceController.scala

```diff
@@ -154,9 +154,9 @@ object BalanceController {
           in.items.head.paymentAmount.getOrElse(logBigDec(0, "payment amount")),
           in.items.head.paymentLot.getOrElse(randomNumbers(10, "payment lot")),
           in.items.head.paymentLotItem.getOrElse(randomNumbers(10, "payment lot item"))).pure[List]
-        case _           => Unknown(dueDate(in), in.mainType.getOrElse("Unknown"), amount(in)).pure[List]
+        case _           => Logger.warn(s"Unknown ${in.mainType} of ${amount(in)} at ${dueDate(in)}"); List.empty
       }
-      case _             => Unknown(dueDate(in), in.mainType.getOrElse("Unknown"), amount(in)).pure[List]
+      case _             => Logger.warn(s"Unknown ${in.mainType} of ${amount(in)} at ${dueDate(in)}"); List.empty
```

Comment:
This will omit unknown SDIL related transactions (such as penalties)
I suggest logging and omitting non-32 transactions and logging and keeping (as Unknown) other type-32 transactions.